src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -17,6 +16,8 @@
 import org.opentripplanner.middleware.otp.response.Leg;
 import org.opentripplanner.middleware.otp.response.OtpResponse;
 import org.opentripplanner.middleware.otp.response.Place;
+import org.opentripplanner.middleware.testUtils.CommonTestUtils;
+import org.opentripplanner.middleware.testUtils.OtpTestUtils;
```

Comment:
The new package name should be all lowercase `org.opentripplanner.middleware.testutils` or   `org.opentripplanner.middleware.test_utils`.

https://stackoverflow.com/a/3179221/915811

Reply:
I vote `testutils`.

Reply:
Changed in 695f54c4ffb08e23144a496d611b999c93c86764.